src/HSE/All.hs

```diff
@@ -183,6 +193,19 @@ data ModuleEx = ModuleEx {
   , ghcAnnotations :: GHC.ApiAnns
 }
 
+-- | The two representations of comments in a tidy little bundle. Used
+-- by 'commentHint'.
+data CommentEx = CommentEx {
+    hseComment :: Comment
+  , ghcComment :: Located GHC.AnnotationComment
+}
+
+-- | Extract a list of all a parsed module's comments.
+comments :: ModuleEx -> [CommentEx]
```

Comment:
Can we get ghcComments promoted to a top-level thing, as I imagine it's going to be useful.

Reply:
And kill comments of course, since there will be no CommentEx

Reply:
✔️